codelabs/grpc-rust-getting-started/start_here/build.rs

```diff
@@ -0,0 +1,18 @@
+fn main() {
+    protobuf_codegen::CodeGen::new()
+        .include("src/routeguide")
+        .inputs(["routeguide.proto"])
+        .output_dir("generated")
+        .compile_only()
+        .unwrap();
+
+    // NOTE: If you were to generate code yourself, you would use the
+    // command below using tonic_protobuf_build's CodeGen.
```

Comment:
nit: I think we should also mention that the above command should to be removed.